build/afterSignHook.macos.js

```
@@ -3,41 +3,27 @@ const path = require("path");
 const electronNotarize = require("electron-notarize");
 const electronBuilderConfig = require("../electron-builder.json");
 
-const { APPLE_TEAM_PROVIDER_ID, APPLE_API_KEY, APPLE_ISSUER_ID } = process.env;
-
 module.exports = async function (params) {
   if (process.platform !== "darwin") {
     return;
   }
 
   console.log("afterSign hook triggered", params);
 
-  // Bail early if this is a fork-caused PR build, which doesn't get
```

why remove the fail early?

---

fixed in a57c7ae